package.json

```diff
@@ -15,39 +15,24 @@
     "swizzle": "docusaurus swizzle",
     "clear": "docusaurus clear",
     "deploy": "docusaurus deploy",
-    "docusaurus": "docusaurus"
+    "docusaurus": "docusaurus",
+    "pre-commit": "lint-staged",
+    "prepare": "git config core.hookspath .githooks"
```

Comment:
This is usually not required for git hooks. The problem here is that now I wouldn't know how to disable git hooks on a per repo basis. I usually change the hooks path to something I own and then symlink the hooks that I do find usefull. 

pre-commit hooks are something I will definitelly disable. So if you want to keep this as a pre-commit hook and not something else (e.g. pre-push), please add docs on how to disable this hook for this particular repository.